src/content/ui-api/web/organizations/access-control/index.md

```diff
@@ -0,0 +1,175 @@
+---
+linkTitle: Access control
+title: Access control for organizations in the web user interface
+description: Learn how to use our web UI for organization RBAC to quickly associate users, groups, and service accounts with organizations.
+weight: 20
+menu:
+  main:
+    identifier: orgs-accesscontrol
+    parent: web-orgs
+owner:
+  - https://github.com/orgs/giantswarm/teams/biscuit
+user_questions:
+  - How can I give someone access to an Organization in the web UI?
+  - How can I give someone access to the web UI?
+  - Where can I find the access control UI?
+last_review_date: 2021-04-20
+---
+
+# Access control for organizations in the web user interface
+
+Learn how to quickly grant and revoke access to an organization's resources via the web user interface, using Kubernetes role based access control (RBAC) mechanisms.
+
+{{% organization_mgmt_mapi_disclaimer %}}
+
+Granting access means that the users (when associating users or groups) or programs (when using service accounts) affected will be able to access resources via the management API, including via the web user interface itself.
```

Comment:
```suggestion
Granting access means that the users (when associating users or groups) or programs (when using service accounts) affected will be able to access resources via the Management API, including via the web user interface itself.
```

Reply:
I'll have to get this sorted everywhere. It's not consistent currently. Thanks